_search-plugins/knn/knn-index.md

```diff
@@ -112,9 +114,41 @@ The IVF algorithm requires a training step. To create an index that uses IVF, yo
 data points, but it is [recommended to use more](https://github.com/facebookresearch/faiss/wiki/Guidelines-to-choose-an-index#how-big-is-the-dataset).
 Training data can either the same data that is going to be ingested or a separate set of data.
 
+### Supported Lucene methods
+
+Method Name | Requires Training? | Supported Spaces | Description
+:--- | :--- | :--- | :---
+`hnsw` | false | l2, innerproduct, cosinesimil | Hierarchical proximity graph approach to Approximate k-NN search.
+
+#### HNSW Parameters
```

Comment:
Should "Parameters" be lowercase? I'm unsure if we apply sentence case to Heading 4.